CodeStarterCamp_Week3/person.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  person.swift
+//  CodeStarterCamp_Week3
+//
+//  Created by daekyo on 2023/10/24.
+//
+
+import Foundation
+
+class Person {
+    var money: Int
+    var name: String
+    
+    init(money: Int, name: String) {
+        self.money = money
+        self.name = name
+    }
+    
+    func buyCoffee(coffeeShop: CoffeeShop, coffee: Coffee) {
```

Comment:
API design guideline에 따라 메서드 이름에 중복된 정보를 줄이고 전달인자 레이블을 활용하면 가독성이 더 좋아질 것 같습니다!
```suggestion
    func buy(coffee: Coffee, at coffeeShop: CoffeeShop) {
```

Reply:
제안 주신거 보니 가독성이 확 좋아 지네요.  앞으로 좀 더 가독성 있게 작성 할 수 있도록 노력하겠습니다.